examples/records/records.bal

```diff
@@ -56,9 +56,9 @@ public function main() {
     io:println(peter);
     io:println(john);
 
-    // This adds an additional field not defined in the record type descriptor above.
-    // Note that an attempt to add additional fields to a closed record results in compile errors.
-    // e.g., `peter.address.street = "Palm Grove";` will result in a compile error.
+    // This adds an additional field, which is not defined in the record type descriptor above.
+    // An attempt to add additional fields to a closed record results in compile errors.
+    // E.g., `peter.address.street = "Palm Grove";` will result in a compile error.
```

Comment:
```suggestion
    // e.g., `peter.address.street = "Palm Grove";` will result in a compile error.
```

Reply:
Capital "E" should be used as this is the beginning of a new sentence.